solr/modules/hdfs/src/java/org/apache/solr/hdfs/HdfsDirectoryFactory.java

```diff
@@ -16,8 +16,6 @@
  */
 package org.apache.solr.hdfs;
 
-import static org.apache.hadoop.fs.CommonConfigurationKeysPublic.HADOOP_SECURITY_AUTHENTICATION;
```

Comment:
These changes should be reverted. modules/hdfs is totally separate from hadoop-auth.

Reply:
will do!   Aside from these changes, how does the rest look?